src/index.d.ts

```diff
@@ -56,6 +58,25 @@ export type CreateFuncReq = Readonly<{
 export interface IFunc {
   arn: string
   eventSourceId?: string
+  name?: string
 }
 
 export type LogGroup = Readonly<{ arn: string }>
+
+export type UpdateConsumersCodeRequest = {
+  consumerIds: ConsumerId[]
+  nodeVersion: string
```

Comment:
Added a validation in [utils validator](src/updateCode/util.ts). Since this object is passed in to the service the typing dosen't add much value there. If we add a ci job this could also be a constraint